types/database/global.d.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {Model} from '@nozbe/watermelondb';
+export default class Global extends Model {
+    static table: string;
+    name: string;
+    value: Object;
```

Comment:
Perhaps we need a broader definition instead of just object? Also I can see us storing just strings or numbers as well

Reply:
I suggest we leave it a temporary type ( string[] ) and update it as we go along.  I have added a todo comment on this file.